doc_build/BUILD

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//:version.bzl", "version")
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+
+package(
+    default_visibility = ["//visibility:private"],
+)
+
+# HOW TO:
+#   bazel build //doc_build:reference
+#   cp bazel-bin/doc_build/reference.md docs/latest.md
+#   git commit -m 'update docs' docs/latest.md
```

Comment:
You might be able to obviate the `cp` step if you use `pkg_install` and set the outgoing permissions accordingly.

Might require updating `pkg_install` for this to work, though.

Reply:
Phase 2 or 3, or 4 :)